immuneML/encodings/pdbencoding/PDBEncoder.py

```diff
@@ -0,0 +1,250 @@
+import matplotlib.colors
+import numpy
+from Bio.PDB import PDBParser
+from immuneML.data_model.encoded_data.EncodedData import EncodedData
+
+from immuneML.data_model.dataset.PDBDataset import PDBDataset
+from immuneML.encodings.DatasetEncoder import DatasetEncoder
+from immuneML.encodings.EncoderParams import EncoderParams
+import numpy as np
+from immuneML.data_model.receptor.RegionType import RegionType
+
+
+class PDBEncoder(DatasetEncoder):
```

Comment:
That was the initial idea, but Python doesn't allow class names that begins with a number. Could we do PDBDistanceMatrixEncoder? Since its only PDB files that will be supported ?

Reply:
PDBDistanceMatrixEncoder is good also! it's specific enough because it mentions the distance matrix part